homeassistant/helpers/config_validation.py

```diff
@@ -1599,7 +1599,7 @@ def determine_script_action(action: dict[str, Any]) -> str:
     if CONF_WAIT_TEMPLATE in action:
         return SCRIPT_ACTION_WAIT_TEMPLATE
 
-    if CONF_CONDITION in action:
+    if any(key in action for key in (CONF_CONDITION, "and", "or", "not")):
```

Comment:
This means that none of the schemas below can have `and`, `or`, `not` as config parameters or else it will be marked as a condition.

Reply:
Yes 🤷 

As they are logical operators already present in conditions now, I think that is OK. There are no current conflicts and worst case, if needed, the logic determining this can be extended/refined in the future to support cases otherwise.